arcgis-ios-sdk-samples/Scenes/Animate 3D graphic/Animate3DGraphicViewController.swift

```diff
@@ -186,6 +184,7 @@ class Animate3DGraphicViewController: UIViewController {
                 //create a frame object for each line
                 frames = lines.map { (line) -> Frame in
                     let details = line.components(separatedBy: ",")
+                    precondition(details.count == 6)
```

Comment:
Just a little sanity check. Ideally we'd convert the file to something more easily parsable, like json. Then we could model it with a decodable type.